### PR TITLE
Fix indent bugs

### PR DESCRIPTION
**Pull request by khanhhuy**
Fix 2 indentation bugs

- The comma is brought to the commented line cause invalid syntax
```sql
select 
  1 as a
  -- comment,
  2 as b
```

- Space is added between `#` and table name
```sql
select {{ # table1 }}
```

https://app.asana.com/0/1112651739443366/1169104230960809

### CODE REVIEW

**Comment by khanhhuy** [comment]
Anh @9diov please help me review this PR nha

File: dist/sql-formatter.js [hunk 1]
```diff
@@ -516,7 +522,7 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	        this.WHITESPACE_REGEX = /^(\s+)/;
 	        this.NUMBER_REGEX = /^((-\s*)?[0-9]+(\.[0-9]+)?|0x[0-9a-fA-F]+|0b[01]+)\b/;
-	        this.OPERATOR_REGEX = /^(!=|<>|==|<=|>=|!<|!>|\|\||::|->>|->|\{\{\{|\}\}\}|\{\{#\w+\s*\}\}|\{\{|\}\}|.)/;
+	        this.OPERATOR_REGEX = /^(!=|<>|==|<=|>=|!<|!>|\|\||::|->>|->|\{\{\{|\}\}\}|\{\{|\}\}|.)/;
```
Comment (9diov) [inline dist/sql-formatter.js:525]: This will break 2.0 code such as  `{#if`

**Comment by khanhhuy** [comment]
Anh @9diov I've just fixed the line comment issue for `#`
Please help me review again

File: dist/sql-formatter.js (hunk 1 above)
Comment (khanhhuy) [reply dist/sql-formatter.js]: Fixed nha anh

**Pull request merged**